practice-two/src/components/Toast/Toast.jsx

```diff
@@ -40,7 +40,7 @@ Toast.propTypes = {
 };
 
 Toast.defaultProps = {
-  message: MESSAGES.SUCCESSES.ADD_SUCCESSED,
+  message: MESSAGES.GET.SUCCESSES.ADD_SUCCESSED,
 };
 
 export default Toast;
```

Comment:
check re-render Toast message

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/2b52d41b-9c14-4f32-b3f6-c4c4f6b83013)
I used memo for Toast component for avoiding re-render